airdrop/application/services/user_registration_services.py

```diff
@@ -32,7 +82,7 @@ def register(self, inputs):
             address = inputs["address"].lower()
             signature = inputs["signature"]
 
-            verify_signature(airdrop_id, airdrop_window_id, address, signature)
+            # verify_signature(airdrop_id, airdrop_window_id, address, signature)
```

Comment:
this needs to be in scope for demo , why have we commented this  ?

Reply:
Fixed in the latest commit

Reply:
dont see it yet  ?